system.go

```diff
@@ -23,16 +23,41 @@ type SystemConnection struct {
 	TotalCount int      `json:"totalCount" graphql:"-"`
 }
 
-type SystemInput struct {
-	Name        *string          `json:"name,omitempty"`
-	Description *string          `json:"description,omitempty"`
-	Owner       *ID              `json:"ownerId,omitempty" yaml:"ownerId,omitempty"`
-	Parent      *IdentifierInput `json:"parent,omitempty" yaml:"parent,omitempty"`
-	Note        *string          `json:"note,omitempty"`
-}
-
 func (s *SystemId) GetTags(client *Client, variables *PayloadVariables) (*TagConnection, error) {
```

Comment:
`TagConnection` should always be the same, can we generate this function? Or better yet just not provide this function and rely on tags.go instead? Would cut down on circular dependencies too, I think.